src/d3-ext/profile.js

```diff
@@ -58,13 +62,13 @@ ngeo.profile = function() {
    * @type {function(Object)}
    * Hover callback function.
    */
-  var hoverCallback = goog.nullFunction;
+  var hoverCallback = options.hoverCallback || goog.nullFunction;
```

Comment:
I disagree.

1. I don't think

 ```js
   var hoverCallback = goog.isDef(options.hoverCallback) ? options.hoverCallback : goog.nullFunction;
 ```

 is harder to read than

 ```js
   var hoverCallback = options.hoverCallback || goog.nullFunction;
 ```

 The first expression is even more explicit, because it makes it clear that the test is about whether `hoverCallback` is defined in the `options` object.

2. It's easier to explain to people that they should **always** use `goog.isDef(options.theOption)` to test if `theOption` is defined in the options.

 I don't want to have to tell people that they can use `||` when dealing with objects, and `goog.isDef` when dealing with number, strings and booleans. I prefer systematic approaches.